.eslintrc.json

```
@@ -0,0 +1,36 @@
+{
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es2021": true
+  },
+  "extends": [
+    "airbnb-base",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:unicorn/recommended",
+    "plugin:sonarjs/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "plugins": ["@typescript-eslint", "unicorn", "sonarjs"],
+  "rules": {
+    "linebreak-style": "off",
+    "no-console": "warn",
+    "comma-dangle": "off",
+    "no-unused-vars": "off",
+    "indent": "off",
+    "object-curly-newline": "off",
+    "guard-for-in": "off",
+    "operator-linebreak": "off",
+    "no-loop-func": "off",
+    "no-restricted-syntax": "off",
+    "arrow-parens": "off",
+    "quotes": "off",
```

These rules are important and you can't switch them off

```suggestion
    "object-curly-newline": "off",
    "operator-linebreak": "off",
    "no-restricted-syntax": "off",
    "arrow-parens": "off",
```